kickstart.ks.j2

```diff
@@ -0,0 +1,29 @@
+ostreesetup --osname=rhel --url=file:///run/install/repo/ostree/repo --ref=rhel/9/x86_64/edge --nogpg --remote=edge
```

Comment:
RHEL "9" can be a variable. We might not have one explicitly. But we can create a const for it for now.